gobblin-metrics/build.gradle

```diff
@@ -85,7 +85,7 @@ artifacts {
 
 test {
     useTestNG () {
-      excludeGroups 'ignore'
+      excludeGroups 'ignore', 'performance'
```

Comment:
Why put this code into TestNg tests when we are going to exclude them from the build anyway? I think it would only make sense to make these tests if we make them fail if the performance numbers exceed a certain value, although even though approach isn't always reliable especially on shared build systems such as Travis.

It may make more sense to move these into a separate package called `gobblin.metrics.performance.eval` (or something like that), and have a shell script to build and generate a report on demand.